<!DOCTYPE html>
<html lang="en">
<head>
     <!--script src="jquery.js"></script-->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="bootstrap-3.0.3/docs-assets/ico/favicon.png">

    <title>MultiCRISPR server</title>

    <!-- Bootstrap core CSS -->
    <link href="bootstrap-3.0.3/dist/css/bootstrap.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="bootstrap-3.0.3/examples/navbar/navbar.css" rel="stylesheet">

    <link rel="stylesheet" type="text/css" href="http://multicrispr.tau.ac.il/css/multicrispr.css">

    <!-- Just for debugging purposes. Don't actually copy this line! -->
    <!--[if lt IE 9]><script src="../../docs-assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
      <script src="https://oss.maxcdn.com/libs/respond.js/1.3.0/respond.min.js"></script>

    <![endif]-->


    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-74752822-1', 'auto');
  ga('send', 'pageview');

</script>

</head>

<body>
    <div class="container" style="background-color:#FFFFFF;">

        <!-- Static navbar -->
        <div class="navbar navbar-default" role="navigation" >

            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <!--a class="navbar-brand" href="#">CLUMPAK</a-->
            </div>
            
            <div class="navbar-collapse collapse">
                <ul class="nav navbar-nav">
                    <li><a href="index.html">Main</a></li>
                    <li><a href="help.html">Help</a></li>
                </ul>
           
            </div>
        </div>     <!-- Static navbar -->

        <div class="row">
          <div class="col-md-10" align=center>
            <img src="CRIPSys.PNG" width="400" height="100" >
          </div>
          <br><br><br><br><br><br>
          <h3>A tool for the optimal design of sgRNAs for simultaneous cleavage of multiple genes.</h3>
        </div>
        
        <div class="jumbotron">
		<p id="description">
          <br>Given a set of genomic sequences as input, CRISPys designs the most promising sgRNAs to target all with high efficiency.
          The webserver proposes four design strategies: 
            <ol type="I">
			<br><li>A single sgRNA that could best target the entire gene set <br><font size="3">This strategy can be executed by selecting the "Consider all genes" optimization criterion. (default)</font></li>
			<br><li>A single sgRNA optimized to target most of the genes, each gene with a score above a given threshold Î© <br><font size="3">This strategy can be executed by selecting the "Consider only genes above a threshold" optimization criterion. </font></li> 
			<br><li>Multiple sgRNAs, each directed towards subgroup of homologous genes<br><font size="3">This strategy can be executed by confirming the "Consider gene homology" button. </font></li>
			<br><li>The minimal set of sgRNAs that could target the entire gene set with high efficiency <br><font size="3">Automatically computed for each execution of strategy II.</font></li>
            </ol> 
          </p>
          <br>
          <br>
          <h2 style="text-align:center;"> <u><b> Usage options</b></u></h2><br>
          <p id="scoring function"> <u><b>The scoring function</b></u><br>
            specifies the estimated targeting efficiency of a given sgRNA to a given genomic site. The site currently supports three scoring functions:
            <ul>
              <li>The CFD score [1]<br></li>
              <li><a href = "http://crispr.mit.edu/about" target="_blank">Optimized CRISPR Design</a> [2]</li>
              <li><a href = https://crispr.cos.uni-heidelberg.de/" target="_blank">CCTop</a> [3]<br><font size="3">The CCTop score is divided by -224 (the minimal potential score) to provide a score between [0,1]. </font></li>
            </ul>
            <br>Additional scoring functions may be integrated upon request.
			<br>
			<div align="center"><figure>
  <img style="max-width: 1000px; height: auto; " src="functions_distribution.png" >
</figure>
			<font size="3"><i>The computed scores and distribution of the cleavage efficiencies (#reads, x axis) reported in Tsai et al. [4]  for targets of the sgRNAs that were applied to U2OS cells. <br>
			(a) The frequencies of the binned #reads. (b-d) The computed scores by the CFD score [1], CCTop [2], and Optimized CRISPR Design [3].</i></font>
          </div>
		  </p><br><br>
          <p id="optimization criterion"><u><b>Optimization criterion</b></u><br></p>
			<p>
          
            CRISPys proposes two criteria for the optimization of sgRNA design:
            <ol>
              <li> The sum of scores across the input sequences (calculated by the selected scoring function). </li>
              <li> The number of sequences that could be cleaved with a score greater than a chosen threshold value. Setting this threshold to 0.0 will result in all input genes, while setting it to 1.0 will practically search for the sgRNA that perfectly matches the largest number of genes. We recommend the thresholds of 0.45, which represents the average CFD score of the targets between the 90th and 95th percentiles in the data by Tsai et al. [4] (see above).</li>
             </ol><br>
			 </p>
          <p id="consider homology">
          <b><u>Consider gene homology</u></b><br></p>
			<p>
		  Enabling this option would execute design strategy III. 
			<b>For selection of proper thresholds, please see the distribution of the scoring functions <a href="#scoring function">presented above</a>. </b></p>
			<p>Dividing the input genes set into smaller groups according to sequence similarity would increase the flexibility of the screen and allow a more focused experimental design. To this end, this strategy recursively splits the input genes into homologous subgroups and generates potential sgRNAs for each of them. 
          </p>
          <p>
            Practically, up to 10 sgRNAs will be designed for each subgroup of size two to nine. </p>

          <br>
          <br>
          <h2 style="text-align:center;"> <u><b>Results Page:</b></u></h2>
          <p>The results for your run are denoted in a table, sorted by the optimal score of the sgRNAs.</p>
          <p>
            Columns:
            <ul>
              <li>sgRNA: the potential sgRNA designed by CRISPys.</li>
              <li>score: sum of cleavage scores for the genes targeted by the designed sgRNA (Genes scores).</li>
              <li>Genes, Genes scores, Target site: list of genes targeted by the sgRNA, with their cleavage score (calculated by the chosen scoring function), and the target site in the sequence.<br>Lower-case letters represent mismatches to the sgRNA. The first and default row holds the gene with highest cleavage score. Press the "+" sign to see the full list of genes</li>
            </ul>
          </p>
          <p>
            When the "Consider gene homology" is enabled, every section of the table corresponds to a homologous genes subgroup as specified by the internal nodes of the constructed genes tree. The name of the subgroup and the list of genes are given in the header of each section.
          </p>
		  <p>
            Often multiple sgRNAs are designed for a single subset of targets. The website presents only the best sgRNA for every subset of targets by default. You may see all possible designs by unfiltering the table.
          </p>
		  
		  <br><br><br>
		  
		  <b><u>References:</u></b><br>
		  [1] Doench, John G., et al. "Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9." Nature biotechnology 34.2 (2016): 184. <br>
		  [2] Zhang Lab, Optimized CRISPR Design, Mit. (2013) 2013. <a href = "http://crispr.mit.edu/about" target="_blank">http://crispr.mit.edu/.</a> <br>
		  [3] M. Stemmer, T. Thumberger, M. del Sol Keyer, J. Wittbrodt, J.L. Mateo, <a href = "https://crispr.cos.uni-heidelberg.de/" target="_blank">CCTop</a>: An Intuitive, Flexible and Reliable CRISPR/Cas9 Target Prediction Tool, PLoS One. 10 (2015) e0124633. doi:10.1371/journal.pone.0124633.<br>
		  [4] Tsai, Shengdar Q., et al. "GUIDE-seq enables genome-wide profiling of off-target cleavage by CRISPR-Cas nucleases." Nature biotechnology 33.2 (2015): 187-197.
		  
		  
		  <br><br><br><br><br>
       
        </div>      <!-- jumbotron -->
 <!--       
        <div class="row">
          <div class="col-md-10" align=center>
            <font size="5" >
            Acknowledgements:
            </font>
        </div>
-->        
    </div> <!-- /container -->
    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
    <script src="bootstrap-3.0.3/dist/js/bootstrap.min.js"></script>
	<!--script src="/js/googleAnalytics.js"></script-->
   
	
</body>
<footer>

</footer>
</html>